src/extension.ts

```diff
@@ -25,33 +25,41 @@ export function start(context: theia.PluginContext): void {
     mocha.useColors(true);
 
     const e = (c: any) => console.log(c);
-
     ncp(context.extensionPath, '/projects/Che-Java-Tests', (err: any) => {
         if (err) {
             return console.error(err);
         }
+        const getSrcDocPath = path.resolve('/projects/Che-Java-Tests/testWorkspace/src/main/java/org/my/sample', 'MyHelloText.java');
```

Comment:
should probably use `theia.workspace.workspaceFolders` (if it's not inside /projects) or other VS code API to grab MyHelloText.java like (workspace / findFiles or something else, etc) ?

Reply:
replaced with findFiles